sim/pkg/serve/index.html

```diff
@@ -64,30 +64,49 @@ <h1 class="title">Skenario</h1>
             </div>
             <div class="field is-horizontal">
                 <div class="field-label is-normal">
-                    <label class="label" for="launchDelay">Replica Launch Delay (seconds)</label>
+                    <label class="label" for="launchDelay">Pod Launch Delay (seconds)</label>
                 </div>
                 <div class="control">
                     <input type="number" style="width: 5em" id="launchDelay" value="5" min="0.01" step="0.1"/>
                 </div>
             </div>
             <div class="field is-horizontal">
                 <div class="field-label is-normal">
-                    <label class="label" for="terminateDelay">Replica Terminate Delay (seconds)</label>
+                    <label class="label" for="terminateDelay">Pod Terminate Delay (seconds)</label>
                 </div>
                 <div class="control">
                     <input type="number" style="width: 5em" id="terminateDelay" value="1" min="0.01" step="0.1"/>
                 </div>
             </div>
             <div class="field is-horizontal">
                 <div class="field-label is-normal">
-                    <label class="label" for="tickInterval">Tick Interval (seconds)</label>
+                    <label class="label" for="tickInterval">Resync Period (seconds)</label>
                 </div>
                 <div class="control">
-                    <input type="number" style="width: 5em" id="tickInterval" value="2" min="1" step="1"/>
+                    <input type="number" style="width: 5em" id="tickInterval" value="15" min="1" step="1"/>
                 </div>
             </div>
 
             <hr>
+            <div class="field is-horizontal">
+                <div class="field-label is-normal">
+                    <label class="label" for="hpa">HPA</label>
+                </div>
+                <div class="control">
+                    <input type="checkbox" name="hpa" onclick="showObject('hpaObject', 'hpa')" style="width: 5em"
+                           id="hpa" checked/>
+                </div>
+            </div>
+            <div id="hpaObject" class="field is-horizontal">
+                <div class="field-label is-normal">
+                    <label class="label" for="hpaYaml">HPA Object (YAML)</label>
+                </div>
+                <div class="control">
+                    <textarea style="width: 15em; height: 30em" id="hpaYaml" min="1" step="1"></textarea>
```

Comment:
Be sure to populate this with a default HPA configuration.  E.g. the hard coded YAML object previously in the binary.

Reply:
Fixed it. See commit b43ddbec6158fa15b687488a9e3c433a62d4e69f